sdk/environments/helpers.go

```diff
@@ -25,6 +25,16 @@ func ApiManagementAPI(domainSuffix string) Api {
 	}
 }
 
+func AppConfigurationAPI(domainSuffix string) Api {
+	return &ApiEndpoint{
+		domainSuffix:       pointer.To(domainSuffix),
+		endpoint:           nil,
+		appId:              pointer.To(apiManagementAppId),
+		name:               "ApiManagement",
```

Comment:
can we update these to be:

```suggestion
		appId:              pointer.To(appConfigurationAppId),
		name:               "AzureAppConfiguration",
```

Reply:
Sorry for that, should have seen that earlier.
Already fixed :)